config/ApplicationUnitTest.cmake

```diff
@@ -265,7 +265,7 @@ macro( aut_register_test )
   # Set arguments that don't change between python and CMake driver
   set (SHARED_ARGUMENTS
       -DAPP=${aut_APP}
-      -DARGVALUE=${argvalue}
+      -DARGVALUE=_av_${argvalue}_av_
```

Comment:
@alexrlongne Can you think of a better way to do this.  I want to capture the string `-a -b -c` as a single argument.  I couldn't figure out how to make that work for both CMake and Python regex, so I added `_av_` as begin() and end() delimiters.

Reply:
For python I can do something like this:
```
 arg = re.compile("(-[A-Za-z0-9]*)"
 my_str = "-option1 -option2"
 arg.findall(my_str)
```

and it returns a nice list:
```
['-option1', '-option2']
```

Let me try a few things in CMake and see if it has something nice like that.

Is there a simple interactive shell for cmake?  :)

Reply:
Resolving this for now, we can revisit regexes later